homeassistant/components/microsoft_face/__init__.py

```diff
@@ -80,11 +81,15 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
         logging.getLogger(__name__), DOMAIN, hass
     )
     entities: dict[str, MicrosoftFaceGroupEntity] = {}
+    domain_config: dict[str, Any] = config[DOMAIN]
+    azure_region: str = domain_config[CONF_AZURE_REGION]
+    api_key: str = domain_config[CONF_API_KEY]
+    timeout: int = domain_config[CONF_TIMEOUT]
```

Comment:
Investigation showed these three were not optional (they have default values in the schema) so I removed `.get`